torch/utils/_content_store.py

```diff
@@ -53,7 +53,7 @@ def lazy_compile(**compile_kwargs):
     def decorate_fn(fn):
         @functools.wraps(fn)
         def compile_hook(*args, **kwargs):
-            compiled_fn = torch.compile(fn, **compile_kwargs)
+            compiled_fn = torch.compiler.enable(torch.compile(fn, **compile_kwargs))
```

Comment:
This is interesting, where the default behavior you want for disable interaction is different for "I'm just using torch.compile to enable compilation for a single operator with dynamic shapes". Maybe we need to deify this into a real API / give it a different name so it's more obviously different from compile?

Reply:
Resolved to not add a new API for this.